conf/messages.en

```diff
@@ -202,3 +205,19 @@ confirmCancellation.error.required = Select yes if you want to cancel this depar
 unauthorisedWithGroupAccess.title = You cannot use this service
 unauthorisedWithGroupAccess.heading = You cannot use this service
 unauthorisedWithGroupAccess.p = If you already have access, you must contact your group administrator to update your access rights.
+
+departure.drafts.dashboard.title = Draft departure declarations
+departure.drafts.dashboard.heading = Draft departure declarations
+departure.drafts.dashboard.search = Search by local reference number (LRN)
+
+departure.drafts.dashboard.table.heading.hidden = Draft departure table
+departure.drafts.dashboard.table.lrn = Local reference number (LRN)
+departure.drafts.dashboard.table.daysToComplete = Days to complete
+departure.drafts.dashboard.table.daysToComplete.hidden = for {0}
+departure.drafts.dashboard.table.action.delete = Delete
+departure.drafts.dashboard.table.delete.action.hidden = Actions
+departure.drafts.dashboard.table.action.delete.hidden = Local Reference Number (LRN) {0}
+
+departure.drafts.dashboard.table.panel.title = Create a new departure declaration
+departure.drafts.dashboard.table.panel.text = You have 30 days from the first time you save a declaration to complete it.
```

Comment:
Is this content a little ambiguous? How does one “save” a declaration? I would have thought “30 days from starting the declaration to complete it” would perhaps be clearer? Question for Gavin